devtoolsNG/xen/elements/inspector/smoke/index.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
Add copyright?

Reply:
Not required for HTML, iiuc.

Reply:
Hrm, but there's embedded js, isn't there. :(